stellar-multitenant/pkg/cli/migrate.go

```diff
@@ -83,16 +83,23 @@ func MigrateCmd(databaseFlagName string) *cobra.Command {
 }
 
 func runMigration(databaseURL string, dir migrate.MigrationDirection, count int) error {
-	numMigrationsRun, err := db.Migrate(databaseURL, dir, count, migrations.FS, db.StellarMultiTenantMigrationsTableName)
+	numMigrationsRun, err := db.Migrate(databaseURL, dir, count, adminMigrations.FS, db.StellarAdminMigrationsTableName)
 	if err != nil {
 		return fmt.Errorf("running migrations: %w", err)
 	}
 
 	if numMigrationsRun == 0 {
 		log.Info("No migrations applied.")
 	} else {
-		log.Infof("Successfully applied %d migrations.", numMigrationsRun)
+		log.Infof("Successfully applied %d migrations %s.", numMigrationsRun, migrationDirectionStr(dir))
```

Comment:
Why not only cast to string?

```suggestion
		log.Infof("Successfully applied %d migrations %s.", numMigrationsRun, string(dir))
```

Reply:
If I use `string(dir)`, the resulting value would be either `0`or `1`:

https://github.com/rubenv/sql-migrate/blob/9f03b5b6c963a717b2b0554dd32246fc8ac342d0/migrate.go#L27-L30